docs/ops/deployment/native_kubernetes.md

```diff
@@ -170,6 +170,34 @@ appender.console.layout.pattern = %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p %-60c %x - %m
 
 If the pod is running, you can use `kubectl exec -it <PodName> bash` to tunnel in and view the logs or debug the process.
 
+## Flink Kubernetes Application
+
+### Start Flink Application
+
+Use the following command to start a Flink application.
+{% highlight bash %}
+$ ./bin/flink run-application -p 8 -t kubernetes-application \
+  -Dkubernetes.cluster-id=<ClusterId> \
+  -Dtaskmanager.memory.process.size=4096m \
+  -Dkubernetes.taskmanager.cpu=2 \
+  -Dtaskmanager.numberOfTaskSlots=4 \
+  -Dkubernetes.container.image=<custom_image_name> \
+  local:///opt/flink/examples/streaming/WindowJoin.jar
+{% endhighlight %}
+
+Note: When deploying a Flink application, the user main code will be executed on Flink Master in the cluster. So the user jars and dependencies should already exist in the image. Follow the [how to customize the Flink Docker image](docker.html#customize-flink-image) to build a customized image baked in user jars on top of Flink official image.
```

Comment:
This needs to go on line 176 above the bash command. I'd also like to show a minimal dockerfile example. 

```suggestion
Application mode allows users to create a single image containing their Job and the Flink runtime, which will automatically create and destroy cluster components as needed. The Flink community provides base docker images [customized](docker.html#customize-flink-image) for any use case.

{% highlight dockerfile %}
FROM flink
COPY /target/my-flink-job-*.jar $FLINK_HOME/usrlib/my-flink-job.jar
{% endhighlight %}
```